corehq/apps/hqwebapp/views.py

```diff
@@ -277,16 +277,25 @@ def server_up(req):
 
     failed = False
     message = ['Problems with HQ (%s):' % os.uname()[1]]
-    for check, check_info in checkers.items():
-        if check_info['always_check'] or req.GET.get(check, None) is not None:
-            try:
-                status = check_info['check_func']()
-            except Exception:
-                # Don't display the exception message
-                status = checks.ServiceStatus(False, "{} has issues".format(check))
-            if not status.success:
-                failed = True
-                message.append(status.msg)
+    only = req.GET.get('only', None)
```

Comment:
Took me a bit to figure out the difference between `serverup.txt?heartbeat` and `serverup.txt?only=heartbeat`. Would be nice if the docstring were updated to note that difference.

Reply:
https://github.com/dimagi/commcare-hq/pull/19889